src/builders/CommandStep.ts

```diff
@@ -8,6 +8,7 @@ import {SkipBuilder, SkipHelper} from './helpers/skip';
 import {PluginBuilder} from './PluginBuilder';
 import {CommandStepSchema, PluginSchema, StepDependsOn} from '../schema';
 import {isPluginBuilder} from './isPluginBuilder';
+import {AgentsHelper} from './helpers/agents';
 
 export class CommandStep
   implements
```

Comment:
Can we implement the `AgentsBuilder` too 🙏

Reply:
✅